content/guides-tutorials/safety.md

```diff
@@ -0,0 +1,103 @@
+---
+title: data safety and workflow
+id: safety
+weight: 80
+draft: false
+author: "people"
+---
+
+
+# Concepts
+
+darktable is a non-destructive editor. It means the original RAW files will never ever be changed by the software, even to change their names. They are archived as-is.
```

Comment:
I wonder if the word "archived" might be confusing here because the word suggests darktable "is archiving files" when the intention is to express the opposite. I'm not sure of an appropriate alternative. Perhaps "The files are read by darktable from their initial import directory" or something.